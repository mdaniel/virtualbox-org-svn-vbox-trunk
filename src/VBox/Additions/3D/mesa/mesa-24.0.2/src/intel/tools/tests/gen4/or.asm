(+f0.0) or(8)   g9<1>.wUD       g9<4>.wUD       0x00000040UD    { align16 };
or(8)           g13<1>.xUD      g13<4>.xUD      g14<4>.xUD      { align16 };
or(8)           g3<1>UD         g3<8,8,1>UD     g5<8,8,1>UD     { align1 };
or(16)          g12<1>UD        g14<8,8,1>UD    g20<8,8,1>UD    { align1 compr };
(+f0.0) or(16)  g12<1>UD        g12<8,8,1>UD    0x3f800000UD    { align1 compr };
or(8)           g8<1>.wUD       g11<4>.xUD      g12<4>.xUD      { align16 NoDDChk };
