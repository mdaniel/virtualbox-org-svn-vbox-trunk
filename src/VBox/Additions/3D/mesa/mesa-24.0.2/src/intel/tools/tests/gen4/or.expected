06 01 61 00 21 0c 28 21 2f 01 6f 00 40 00 00 00
06 01 60 00 21 04 a1 21 a0 01 60 00 c0 01 60 00
06 00 60 00 21 04 60 20 60 00 8d 00 a0 00 8d 00
06 20 80 00 21 04 80 21 c0 01 8d 00 80 02 8d 00
06 20 81 00 21 0c 80 21 80 01 8d 00 00 00 80 3f
06 09 60 00 21 04 08 21 60 01 60 00 80 01 60 00
